cmd/neofs-cli/modules/container.go

```diff
@@ -28,4 +215,119 @@ func init() {
 	// Cobra supports local flags which will only run when this command
 	// is called directly, e.g.:
 	// containerCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+
+	listContainersCmd.Flags().StringVar(&containerOwner, "owner", "", "owner of containers (omit to use owner from private key)")
+
+	createContainerCmd.Flags().StringVar(&containerACL, "basic-acl", "private",
+		"hex encoded basic ACL value or keywords 'public', 'private', 'readonly'")
+	createContainerCmd.Flags().StringVarP(&containerPolicy, "policy", "p", "",
+		"QL-encoded or JSON-encoded placement policy or path to file with it")
+	createContainerCmd.Flags().StringArrayVarP(&containerAttributes, "attribute", "a", nil,
+		"colon separated pair of container attribute key and value, e.g. `target:cats`")
+	createContainerCmd.Flags().StringVarP(&containerNonce, "nonce", "n", "", "UUIDv4 nonce value for container")
+	createContainerCmd.Flags().BoolVar(&containerAwait, "await", false, "block execution until container will be persisted")
+
+	deleteContainerCmd.Flags().StringVar(&containerID, "cid", "", "container ID")
+	deleteContainerCmd.Flags().BoolVar(&containerAwait, "await", false, "block execution until container will be removed")
```

Comment:
block execution until container is removed

Reply:
Fixed.